src/containers/List/ListWrapper.tsx

```diff
@@ -8,18 +9,20 @@ import { useLanguageProvider } from '../../contexts/Localisation'
 import { findUserRole, checkExistingUserRole } from '../../utils/helpers/list/findUserRole'
 import { useUserState } from '../../contexts/UserState'
 import { useMapColumnsByRole } from '../../utils/helpers/list/mapColumnsByRole'
-import { ApplicationListRow, ColumnDetails, SortQuery } from '../../utils/types'
+import { ApplicationListRow, ColumnDetails, FilterDefinitions, SortQuery } from '../../utils/types'
 import { USER_ROLES } from '../../utils/data'
 import { Link } from 'react-router-dom'
 import ApplicationsList from '../../components/List/ApplicationsList'
 import PaginationBar from '../../components/List/Pagination'
 import ListFilters from './ListFilters/ListFilters'
 import { useApplicationFilters } from '../../utils/data/applicationFilters'
 import Loading from '../../components/Loading'
+import { usePrefs } from '../../contexts/SystemPrefs'
 
 const ListWrapper: React.FC = () => {
   const { strings } = useLanguageProvider()
-  const APPLICATION_FILTERS = useApplicationFilters()
+  const { preferences } = usePrefs()
+  const APPLICATION_FILTERS = useApplicationFilters(preferences?.defaultListFilters || [])
```

Comment:
I think the fallback should be a couple of the more common filters, not empty array. Cos we want some defaults even on older snapshots and on snapshots that don't have anything defined in prefs.

I'd imagine that if the hard-coded defaults are fine for most cases, adding to the preferences won't be necessary, it's only if they want something different.